### ISSUE TITLE
alm.tools hacked?

### ISSUE THREAD

**Issue by zkf** [alm-tools/alm-tools.github.io#5]
Hi,

Just a heads-up:
There looks to be some issues with the content on alm.tools compared to alm-tools.github.io. It looks like some spammy links and content has been injected into the site.

Take care.

**Comment by basarat** [comment]
Yeah, I let the domain go cause I was wasting money on open source projects e.g. typestyle.io / alm.tools (others that didn't take off). 

Sad to see someone buy it (that would be okay, I didn't want to camp) and abuse it (not okay 😢, notice buyinstagramlikes.io)  :

![image](https://user-images.githubusercontent.com/874898/56252177-009eac00-60fa-11e9-9d79-c2d5d4f15cfa.png)
